src/NuGet.Core/NuGet.Protocol/Converters/ParserConstants.cs

```diff
@@ -63,5 +63,11 @@ public static class JsonProperties
         public const string SigningCertificates = "signingCertificates";
         public const string AllRepositorySigned = "allRepositorySigned";
         public const string Fingerprints = "fingerprints";
+
+        // deprecation properties
+        public const string Deprecation = "deprecation";
+        public const string DeprecationMessage = "message";
+        public const string DeprecationReasons = "reasons";
+        public const string AlternatePackage = "alternatePackage";
```

Comment:
Everywhere you have alternate should be alternative (I realize this might mean changes in the server schema) 

alternate is taking turns. 
alternative is a different option. 
@scottbommarito

Reply:
I think both can be used interchangeably.
https://www.englishgrammar.org/alternate-vs-alternative/

Would prefer not to _alter_ the server schema at this point ;)